src/azure-cli/azure/cli/command_modules/compute_recommender/aaz/latest/compute_recommender/_spot_placement_recommender.py

```diff
@@ -56,21 +59,28 @@ def _build_arguments_schema(cls, *args, **kwargs):
             options=["--availability-zones"],
             arg_group="SpotPlacementScoresInput",
             help="Defines if the scope is zonal or regional.",
+            is_preview=True,
         )
         _args_schema.desired_count = AAZIntArg(
             options=["--desired-count"],
             arg_group="SpotPlacementScoresInput",
             help="Desired instance count per region/zone based on the scope.",
+            required=True,
```

Comment:
Please note that `required=True` will cause breaking changes to the customer's automated scripts. 
Due to the late submission of this PR, you have missed the train of breaking change window for this sprint. I suggest releasing the breaking change part in the next breaking change window (next year's build event sprint)

Reply:
Is there any choice to release it earlier before breaking change release? It is to correct the optional/required parameter. Today, if customers miss those parameter, they will get error from our api. I don't think this change will break the customer's script.

Reply:
Okay, got it. It's not considered a breaking change.

Reply:
@ joal42 It will be released in the next sprint **(12-10)**